ring/ring.go

```diff
@@ -564,12 +571,46 @@ func (r *Ring) countTokens() (map[string]uint32, map[string]uint32) {
 	return numTokens, owned
 }
 
-// updateRingMetrics updates ring metrics.
+// countTokens returns the number of tokens and tokens within the range for each instance.
+func countTokensUtil(ringDesc *Desc) (map[string]uint32, map[string]uint32, int) {
```

Comment:
There is quite a bit of duplication between `Ring.countTokens()` and `countTokensUtil()` now. 
Couldn't `countTokensUtil` replace `Ring.countTokens()`?

Reply:
working on it. Actually Ring.countTokens is only used at one more place in ring/http.go

Reply:
went with different solution